docs/docs/recipes.md

```diff
@@ -56,11 +75,11 @@ You can create pages in Gatsby explicitly by defining React components in `src/p
 
 Routing in Gatsby relies on the `<Link />` component.
 
-### Requirements
+### Prerequisites
```

Comment:
@gillkyle this is the only recipe that needed updating in the existing source, from what I could find.

Reply:
I guess I've started seeing PRs with Requirements in the text, I'll make sure to be aware of this so they get fixed (like in #15252)